package.json

```diff
@@ -43,8 +43,12 @@
     "typescript": "^4.9.5"
   },
   "dependencies": {
+    "@aws-sdk/client-dynamodb": "^3.321.1",
+    "@aws-sdk/lib-dynamodb": "^3.321.1",
+    "@aws-sdk/util-dynamodb": "^3.321.1",
```

Comment:
It looks like util-dynamodb is unused as well.

It's important to be thoughtful when adding npm dependencies. Data breaches via software supply chains (mainly npm packages in our case) are all too common. Plus the more dependencies we have the larger the artifact size which just slows things down. I've seen plenty of other weird issues with third party dependencies. It's best not to include them if we don't really need them.